### ISSUE TITLE
If you take someone's debit card and guess there pin you get full access to there bank account

### ISSUE THREAD

**Issue by Rivallrelyks** [HollywoodHeists/public#177]
![dwight-theoffice](https://user-images.githubusercontent.com/115848318/216871166-c7a55758-cb7f-42d4-bfb4-f1624e7575c2.gif)

**Comment by RockemRobotics** [comment]
I mean... That seems pretty on point for RP and realism.

**Comment by Rivallrelyks** [comment]
the max you can take off someone is 10k, you have access to there full bank account if you take there debit card. and people would just start robbing for debit cards cause once that moneys in the account or given to someone else its untraceable. its to easy to power game the mechanic and would lead to way to many issues

**Comment by ItsTrihawk** [comment]
I think it brinks on the point that you have access to all of it. Rules state you can only rob someone of 10k and can't force them to pull money out, if you get robbed card taken and they get access what's the rule on that? -also was typing this as someone was stating the exact same thing as me-

**Comment by RockemRobotics** [comment]
Yeah but you have to know/guess their PIN. Can you change your PIN?

**Comment by ItsTrihawk** [comment]
Who is to say once you have that card you don't continue to abuse it and pass it around to other people? It's scummy and isn't good for rp imo

**Comment by RockemRobotics** [comment]
> Who is to say once you have that card you don't continue to abuse it and pass it around to other people? It's scummy and isn't good for rp imo

What I'm asking/saying is that if your card gets stolen, you should change your PIN. Or make your PIN not a number that's easy to guess.

**Comment by ItsTrihawk** [comment]
> > Who is to say once you have that card you don't continue to abuse it and pass it around to other people? It's scummy and isn't good for rp imo
> 
> 
> 
> What I'm asking/saying is that if your card gets stolen, you should change your PIN. Or make your PIN not a number that's easy to guess.

I don't carry a card for the exact reason as stated so I don't know

**Comment by QuintenM911** [comment]
All cards are attached to you Citizen ID so it could also be a thing where you get your card canceled through the QBbank and dont have to worry
